src/main/java/org/opendatakit/suitcase/model/ODKCsv.java

```diff
@@ -482,8 +483,16 @@ private String checkForNull(String val) {
       this.attMngr.getListOfRowAttachments(rowId);
 
       if (config.isScanFormatting()) {
-        this.attMngr.downloadAttachments(rowId, true);
-        scanRaw = new ScanJson(this.attMngr.getScanRawJsonStream(rowId));
+          try {
+              this.attMngr.downloadAttachments(rowId, true);
+              scanRaw = new ScanJson(this.attMngr.getScanRawJsonStream(rowId));
+          }
+          catch (JSONException e)
+          {
+              e.printStackTrace();
+              DialogUtils.showError("Scan formatting is not possible for the selected table",true);
+              config.setScanFormatting(false);
```

Comment:
I think it makes more sense to abort the operation instead of selecting a different option. That also avoids adding an extra publicly mutable field to `CsvConfig`.

Is there a way to check if the attachment exists instead of catching the exception?

Reply:
Thanks for your suggestion @linl33 , The table I uploaded did had attachments and the download attachments option was also working fine . I used the geotagger table ( https://github.com/odk-x/suitcase/tree/master/testfiles/dataToUpload )  . Only this scan formatting option is returning an error . In the table metadata fetched I was unable to find any parameter which could determine if scan formatting is possible or not .
I can add code to abort the operation if a JSONException is thrown by this method . Please suggest

Reply:
@linl33 thoughts?

Reply:
What I meant by "attachment exists" is to check if the particular attachment required to apply scan formatting exists. I think it makes more sense to abort the download when a `JSONException` is caught here.